### ISSUE TITLE
free pro not clear in form builder 

### ISSUE THREAD

**Issue by svenl77** [Themekraft/BuddyForms-Members#16]
<img width="1075" alt="screen shot 2018-04-25 at 13 13 54" src="https://user-images.githubusercontent.com/645567/39245071-96ce20d6-488a-11e8-9afb-9d7dbda3899b.png">

**Comment by svenl77** [comment]
<img width="1049" alt="screen shot 2018-05-18 at 10 24 57" src="https://user-images.githubusercontent.com/645567/40227362-363bfc76-5a86-11e8-9d9d-afe173969b95.png">

**Comment by gfirem** [comment]
Where suppose to be showing the pro version tag?

**Comment by svenl77** [comment]
yes, all should be pro labeled. Member type is a pro only... And feel like this worked before. Maybe a bootstrap issue

**Comment by svenl77** [comment]
This is still an issue and confuses the people we need to add a Pro only label same like we do for other forms elements
